src/core/ext/transport/chttp2/transport/ping_callbacks.cc

```diff
@@ -16,13 +16,18 @@
 
 #include "src/core/ext/transport/chttp2/transport/ping_callbacks.h"
 
+#include <inttypes.h>
+
+#include <cstdint>
 #include <utility>
 
 #include "absl/meta/type_traits.h"
 #include "absl/random/distributions.h"
 
 #include <grpc/support/log.h>
 
+grpc_core::TraceFlag grpc_ping_trace(false, "ping");
```

Comment:
Please document the tracer in doc/environment_variables.md.

Reply:
done